apps/sage_intacct/utils.py

```
@@ -97,7 +104,20 @@
+    def is_sync_allowed(self, attribute_type: str, attribute_count: int):
+        """
+        Checks if the sync is allowed
+
+        Returns:
+            bool: True
+        """
+        workspace_created_at = Workspace.objects.get(id=self.workspace_id).created_at
+        if workspace_created_at > datetime(2024, 10, 1) and attribute_count > SYNC_UPPER_LIMIT[attribute_type]:
+            return False
+
+        return True
```

Ruff

Return the negated condition directly

Inline condition

(SIM103)